### ISSUE TITLE
WifiStateChangeReceiver can't work in android 8.0 of Pixel mobile

### ISSUE THREAD

**Issue by tuzhao** [ppareit/swiftp#102]
WifiStateChangeReceiver declared in Manifest.xml will not support receive android.net.wifi.STATE_CHANGE, should use registerReceiver() method.

**Comment by ppareit** [comment]
Yes, another implicit broadcast registered in the android manifest.

**Comment by tuzhao** [comment]
right,the android system have limit receive system broadcast in manifest.xml.

**Comment by ppareit** [comment]
I'll keep this open. This will become a problem for android api 26 and up.
